src/sentry/static/sentry/app/views/organizationDiscover/result/chart.jsx

```diff
@@ -64,21 +65,48 @@ export default class Result extends React.Component {
         });
       }
 
-      result.push({seriesName: key, data: output[key].data});
+      let seriesNameValue = key;
+      if (key.length > 60) {
+        seriesNameValue = key.substring(0, 60) + '...';
+      }
+      result.push({seriesName: seriesNameValue, data: output[key].data});
     }
     return result;
   }
 
+  checkLineChart(chartData) {
+    for (let i = 0; i < chartData.length; i++) {
+      if (chartData[i].data.filter(({value}) => value !== null).length > 1) {
+        return true;
+      }
+    }
+    return false;
+  }
+
   render() {
     const {fields} = this.props.query;
     const {data} = this.props.data;
 
     const chartData = this.getChartData(data, fields);
+    console.log('data', data);
+    console.log('chart data', chartData);
 
     return (
       <div>
-        <LineChart series={chartData} style={{height: 300}} />
-        <BarChart series={chartData} stacked={true} style={{height: 300}} />
+        {this.checkLineChart(chartData) ? (
```

Comment:
Can we rename `checkLineChart` to something more descriptive? i.e. what are we checking LineChart for?

Reply:
it's checking for if there are any series where two or more data points will be graphed.
Maybe renderLineChart?